main.py

```diff
@@ -487,9 +487,16 @@ async def pull(self, ctx):
             self.pity.update({user: {4: 0, 5: 0}})
             self.database.new_pity(user, 0, 0)
 
+        chance5 = 6 #set pity to 0.6% when not soft pity
         pity = False
         self.pity[user][4] += 1
         self.pity[user][5] += 1
+
+        if self.pity[user][5]>75:
+            if self.pity[user][5] == 76:
```

Comment:
1. This logic does not correctly implement the described soft pity system
2. This entire commit can easily be compacted into one line

`if num <= max(300 - 20 * (self.pity[user][5] - 76), 6)`

Reply:
so true